src/main/resources/edu/wpi/first/outlineviewer/MainWindow.fxml

```diff
@@ -19,6 +19,7 @@
         <MenuBar BorderPane.alignment="CENTER">
             <Menu mnemonicParsing="false" text="File" id="fileMenu">
                 <MenuItem onAction="#showPreferences" text="Preferences" id="preferencesMenuButton" accelerator="SHORTCUT+,"/>
+                <MenuItem onAction="#exitProgram" text="Exit" id="exitMenuButton" accelerator="SHORTCUT+x"/>
```

Comment:
Seems like Q would be a more appropriate shortcut.

Reply:
definitely cmd-Q on a mac. Is it also Q on windows for the exit shortcut?